webpack/components/AnsibleHostDetail/components/RolesTab/RolesTable.js

```diff
@@ -27,7 +26,7 @@ const RolesTable = ({
   hostGlobalId,
   canEditHost,
 }) => {
-  const columns = [__('Name'), __('Variables')];
+  const columns = [__('Name'), __('Variables'), __('Source')];
```

Comment:
Maybe source is bit missleading, but I do not have a better word for you, so just pointing out in case someone would have better idea ^_^

Reply:
Yeah, I spent more time on this than I want to admin and still couldn't came up with anything better than `Source`. But I'm open to suggestions and ideas!

Reply:
not sure if it's better, how about `'Inherited from host group'`? 

and later on:
```js
<Td>{ role.inherited ? <CheckIcon /> : null }</Td>
```